### ISSUE TITLE
Not Work on android 9.

### ISSUE THREAD

**Issue by khaclep007** [microg/android_packages_apps_GmsCore#722]
im use custom pixel experience with android version is PIE(9.0) and that cannt installed.

**Comment by Vavun** [comment]
Because you need to remove gapps first
https://github.com/microg/android_packages_apps_GmsCore/wiki/Prerequisites

**Comment by m-khvoinitsky** [comment]
Related question: when should we wait for lineageos for microg 16 builds?

**Comment by khaclep007** [comment]
i install it on vistualXposed,and what i should remove gapps?

**Comment by cRaZy-bisCuiT** [comment]
> Related question: when should we wait for lineageos for microg 16 builds?

They're already done, if a LineageOS 16 official build for your device exists.

Unfortunately there's none for whyred...

**Comment by m-khvoinitsky** [comment]
> They're already done, if a LineageOS 16 official build for your device exists.

Yep, thank you. I've already updated a few days ago. When I asked, there weren't yet any builds.

**Comment by JohnGLFour** [comment]
It works on android 9. Tested with latest lineageos with microg and xposed module.
